### ISSUE TITLE
Gradient Tool: Add Radial Direction Option 

### ISSUE THREAD

**Issue by munirkamal** [WordPress/gutenberg#24049]
Hi,

It would be great if there would be a direction option for radial-gradient type. Here is how it works in Divi Theme.
https://www.loom.com/share/ccd86ba0db8f44b29f9af91ff1fc2959

Thanks

**Comment by ItsJonQ** [comment]
That's interesting! I'm interested to hear what others think

Ultimately, I think the goal is to offer maximum control with minimal UI. That's also the biggest challenge 🙈 .
At the moment, the existing gradient controls need to be visually simplified (in my opinion). I'm not sure where to add this positional feature (as cool as it may be!).

Don't let my thoughts dissuade anything though ❤️

**Comment by munirkamal** [comment]
@ItsJonQ 

Here is an online tool I found that also allows to set directions for the radial gradient.
https://www.css-gradient.com/

**Comment by richtabor** [comment]
> At the moment, the existing gradient controls need to be visually simplified (in my opinion).

I completely agree. 

<img width="276" alt="Screen Shot 2020-07-20 at 2 06 44 PM" src="https://user-images.githubusercontent.com/1813435/87971110-d3764180-ca92-11ea-87aa-0a3f0dc581bb.png">

**Comment by munirkamal** [comment]
@ItsJonQ 

Yeah, I also agree that the control needs some more refinements as there are a few glitches currently in using it. I just felt to add this issue as well while you are working on it, as this may enable more design possibilities in my opinion.

**Comment by mapk** [comment]
Some of the gradient controls are getting updated here: https://github.com/WordPress/gutenberg/pull/23802

**Comment by mtias** [comment]
For reference, this is the design target:

![image](https://user-images.githubusercontent.com/548849/91331012-83457a00-e7ca-11ea-9c04-261357f80b3f.png)

**Comment by draganescu** [comment]
Would a nice matrix control work for the radial center? 

<img width="1434" alt="Screenshot 2023-04-07 at 19 03 41" src="https://user-images.githubusercontent.com/107534/230640678-5060054c-6ca2-4439-b97e-961f8785c918.png">

**Comment by richtabor** [comment]
Maybe, if you think that'd be enough control. I can imagine you'd want more flexibility though, like offsetting the radial out of the visible canvas to make it more subtle. 

How do other web editor tools handle radial gradients?

**Comment by mtias** [comment]
We had a design for this already, no?

**Comment by draganescu** [comment]
> How do other web editor tools handle radial gradients?

From what I've seen they offer fixed options. I think it's because the option in CSS is pretty convoluted.

@mtias I think missed you previous comment :) Either way it'd be great to have it because it allows for many creative background [options.

Here ](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/radial-gradient) we can see we have to account for position of the center of the radial gradient but also extent. How would the direction knob solve this?

**Comment by richtabor** [comment]
It's a bit more "hands on" but here's how Framer (and also Figma) handle more precise control for radial gradients. May be a fun experiment, though I don't think radial direction is a priority. 


https://user-images.githubusercontent.com/1813435/233228996-c3143ca8-9427-4f6a-b514-c093d69e7bec.mp4

**Comment by jasmussen** [comment]
This is fixed!

<img width="622" height="403" alt="Image" src="https://github.com/user-attachments/assets/f65f0185-39f2-4dc0-9aaf-52a2f23d799d" />
